include/linux/mroute.h

```diff
@@ -2,8 +2,10 @@
 #ifndef _UAPI__LINUX_MROUTE_H
 #define _UAPI__LINUX_MROUTE_H
 
+#ifdef GNU_LINUX
```

Comment:
Same problem.

Reply:
If you check other files we have under "include/linux", they all have the same ifdef.